docs/user/agent/node-joiner.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+if [ $# -lt 1 ]; then
+    echo "./node-joiner.sh <pull secret path>"
+    echo "Usage example:"
+    echo "$ ./node-joiner.sh ~/config/my-pull-secret"
+
+    exit 1
+fi
+pullSecret=$1
+
+# Extract the installer image pullspec and release version.
+releaseImage=$(oc get clusterversion version -o=jsonpath='{.status.history[?(@.state == "Completed")].image}')
+nodeJoinerPullspec=$(oc adm release info -a "$pullSecret" --image-for=installer "$releaseImage")
```

Comment:
For cluster operators, CVO sets up a configMap containing the correct image(s) to use. I wonder if we can use that mechanism to save having to get the pull secret.
The tricky part is knowing what operator we can sensibly attach it to, since we can't create a new CVO operator just for this.

Failing that, we should be able to get the pull secret from the cluster.